packages/components/CHANGELOG.md

```diff
@@ -30,6 +30,10 @@
 
 -   `Tabs`: improve focus handling in controlled mode ([#56658](https://github.com/WordPress/gutenberg/pull/56658)).
 
+### Internal
```

Comment:
Noting that there's already an `Internal` section under `Unreleased`, so let's use that instead.

Reply:
Fixed by [8db901f](https://github.com/WordPress/gutenberg/pull/56758/commits/8db901f66b691882be617d2d852e6f96bbdb7746) 👍